core/jobs_test.py

```diff
@@ -75,12 +93,31 @@ def entity_classes_to_map_over(cls):
 
     @staticmethod
     def map(item):
+        """Implements the map function. Must be declared @staticmethod.
+
+        Args:
+            item: *. A single element of type given by entity_class().
+
+        Yields:
+            tuple(str,int). It will yield a integer value.
+        """
         current_class = MockJobManagerTwo
         if current_class.entity_created_before_job_queued(item):
             yield ('sum', 1)
 
     @staticmethod
     def reduce(key, values):
+        """Implements the reduce function. Must be declared @staticmethod.
+
+        Args:
+            key: *. A key value as emitted from the map() fuction, above.
+            values: list(*). A list of all values from all mappers that
+                were tagged with the given key.
+
+        Yields:
+            str. It will yield a string value.
```

Comment:
Please fix the type. Generally, Oppia jobs yield `tuple(str, *)`, where the `*` can change depending on the job.